integration_tests/ipp/container-cups-tls/cupsssl.conf

```diff
@@ -0,0 +1,152 @@
+#
+# Configuration file for the CUPS scheduler.  See "man cupsd.conf" for a
+# complete description of this file.
+#
+
+# Log general information in error_log - change "warn" to "debug"
+# for troubleshooting...
+LogLevel warn
+PageLogFormat
+
+# Deactivate CUPS' internal logrotating, as we provide a better one, especially
+# LogLevel debug2 gets usable now
+MaxLogSize 0
+
+# Only listen for connections from the local machine.
+Listen /var/run/cups/cups.sock
+
+# Show shared printers on the local network.
+Browsing Off
+BrowseLocalProtocols dnssd
+
+# Default authentication type, when authentication is required...
+DefaultAuthType Basic
+
+# Web interface setting...
+WebInterface Yes
```

Comment:
If WebInterface is `No`, does that affect the scan?

Reply:
It shouldn't affect anything. The scan is sending get-printer-attributes requests (and optionally CUPS-specific CUPS-get-printers requests), which don't rely on the web interface. It's mostly that way because I modified as few of the default settings as possible.
It's possible that we could scrape information off of the web interface if it's enabled from all addresses.

Reply:
If the default is on, then it's not a big deal.

A future TODO might be detecting if the web interface is enabled; that could be an interesting metric ("how many publicly-exposed print servers also public expose the web management interface?"), but probably very interesting to implement.